src/main/java/uk/co/ramp/covid/simulation/place/Household.java

```diff
@@ -186,23 +204,33 @@ public void doTesting(Time t) {
         }
     }
 
-    private void moveNeighbour(boolean lockdown) {
+    private void moveNeighbour(Time t, boolean lockdown) {
+        // We only visit neighbours between 9 - 20
+        if (!visitsNeighbourToday || t.getHour() < 8 || t.getHour() >= 19) {
```

Comment:
Great. we should kick these to the parameters file at some point, but looks good for now